code/modules/surgery/organs/augments_arms.dm

```diff
@@ -484,6 +484,11 @@
 	var/disabled = FALSE
 	var/force_when_disabled = 5 //still basically a metal pipe, just hard to move
 
+/obj/item/shield/v1_arm/customised_abstract_text()
+	if(!ishuman(loc))
+		return
+	var/mob/living/carbon/human/owner = loc
+	return "<span class='warning'>[owner.p_their(TRUE)] [owner.l_hand == src ? "left arm" : "right arm"] is covered in metal.</span>\n"
```

Comment:
Feels to me like it'd make sense to move the trailing `\n` out of these checks and just concatenate them to the resulting string when it gets added to `msg`.

Reply:
like this?
```
var/msg = "" //string with checks
return "[msg]\n"
```

Reply:
I think they mean in the proc where its called
`var/returned = "[item.customised_abstract_text()]\n"`

Reply:
alright, added this
not actually like you pointed here because every items has `customised_abstract_text()` proc output but not all abstract items has this text but `\n` is now in examine proc